graph-builder/src/graph.rs

```diff
@@ -214,7 +233,9 @@ pub fn run<'a>(settings: &'a config::AppSettings, state: &State) -> ! {
 
         match serde_json::to_string(&graph) {
             Ok(json) => {
-                *state.json.write().expect("json lock has been poisoned") = json;
+                *state.json.write() = json;
+                *state.live.write() = true;
```

Comment:
I fear that we should signal "live" way before this.
Otherwise once we start having a large number of releases to scrape, we may enter in a restart-loop where the pod takes too much time to become live and is continuously killed and rescheduled.

Reply:
I thought about that too, but we have https://github.com/openshift/cincinnati/blob/40e5956b0fbc5459bc0b762d6cd84a8942f1f75c/dist/openshift/cincinnati.yaml#L81 in place. I'm fine removing that delay and setting live earlier.

Reply:
> I'm fine removing that delay and setting live earlier.

The delay is now 3 seconds the live is set before the scraping loop.